raft/src/main/java/org/apache/kafka/raft/QuorumState.java

```diff
@@ -83,7 +83,8 @@ public int leaderIdOrNil() {
     }
 
     public OptionalLong highWatermark() {
-        return state.highWatermark();
+        OptionalLong optionalLong = state.highWatermark();
```

Comment:
nit: this change doesn't seem very helpful. Perhaps it was made during debugging?

Reply:
Sounds good